.github/workflows/acceptance-tests.yml

```
@@ -0,0 +1,121 @@
+name: Acceptance tests
+
+on:
+  pull_request:
+    types:
+    - opened
+    - edited
+    - synchronize
+    - reopened
+    - closed
+
+jobs:
+  check-deploy:
+    runs-on: ubuntu-20.04
+    outputs:
+      pr-contains-string: ${{ steps.deploy-comment.outputs.pr-contains-string }}
+    steps:
+      - id: deploy-comment
+        uses: SwissDataScienceCenter/renku/actions/check-pr-description@000-deploy-action
+        with:
+          string: /deploy
+          pr_ref: ${{ github.event.number }}
+  deploy-pr:
+    needs: check-deploy
+    if: github.event.action != 'closed' && needs.check-deploy.outputs.pr-contains-string == 'true'
+    runs-on: ubuntu-20.04
+    environment:
+      name: ci-rp-${{ github.event.number }}
+    steps:
+    - name: deploy-pr
+      uses: SwissDataScienceCenter/renku/actions/deploy-renku@000-deploy-action
+      env:
+        DOCKER_PASSWORD: ${{ secrets.RENKU_DOCKER_PASSWORD }}
+        DOCKER_USERNAME: ${{ secrets.RENKU_DOCKER_USERNAME }}
+        GITLAB_TOKEN: ${{ secrets.DEV_GITLAB_TOKEN }}
+        KUBECONFIG: "${{ github.workspace }}/renkubot-kube.config"
+        RANCHER_PROJECT_ID: ${{ secrets.CI_RANCHER_PROJECT }}
+        RENKU_RELEASE: ci-rp-${{ github.event.number }}
+        RENKU_VALUES_FILE: "${{ github.workspace }}/values.yaml"
+        RENKU_VALUES: ${{ secrets.CI_RENKU_VALUES }}
+        RENKUBOT_KUBECONFIG: ${{ secrets.RENKUBOT_DEV_KUBECONFIG }}
+        RENKUBOT_RANCHER_BEARER_TOKEN: ${{ secrets.RENKUBOT_RANCHER_BEARER_TOKEN }}
+        RENKU_BOT_DEV_PASSWORD: ${{ secrets.RENKU_BOT_DEV_PASSWORD }}
+        RENKU_ANONYMOUS_SESSIONS: false
+        RENKU_TESTS_ENABLED: true
+        renku: "@000-deploy-action"
+        renku_core: "@${{ github.head_ref }}"
+    - name: Check existing renkubot comment
+      uses: peter-evans/find-comment@v1
+      id: findcomment
+      with:
+        issue-number: ${{ github.event.pull_request.number }}
+        comment-author: 'RenkuBot'
+    - name: Create comment pre deploy
```

Are you re-posting the link on purpose or did an `if` get lost?
```suggestion
    - name: Create comment pre deploy
      if: steps.findcomment.outputs.comment-id == 0
```